src/modeler/AutoRest.Swagger/Validation/Core/Rule.cs

```diff
@@ -27,6 +27,11 @@ protected Rule()
         public virtual ValidationCategory ValidationCategory => ValidationCategory.None; // !!! implement me and make me abstract !!!
 
         /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public virtual ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.None;
```

Comment:
I'm open to the suggestion, how do you think these 'links' would look like, urls to the repo or relative paths to the corresponding .md file